rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -136,9 +137,9 @@ class AsyncParametersClient
   std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>
   set_parameters(
     std::vector<rclcpp::parameter::ParameterVariant> parameters,
-    std::function<
-      void(std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)
-    > callback = nullptr)
+    std::function<void(
+      std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)> callback =
+    nullptr)
```

Comment:
That is interesting. It does not change the current file at all when I run `ament_uncrustify --reformat` on the file. Can you please provide more information what exactly you did? May be also provide a `vcs export --exact` dump. If our code style checked would behave non-deterministic we should really investigate this further.